### ISSUE TITLE
How do I install/setup plexiptv on PMS mac?

### ISSUE THREAD

**Issue by jtimmyf** [xiaodoudou/PlexIPTV#26]
I see no instructions? This isn't a plugin I don't think?? I'd like to be able to watch & record iptv, vaders, through plex if this is possible.
Thnks

**Comment by xiaodoudou** [comment]
You are right that it is not a Plex plugin. It is a separate small server you
run alongside Plex, which pretends to be an HDHomeRun tuner so Plex will treat
your IPTV subscription as live TV.

On a Mac:

1. Download `PlexIPTV.macos.x64` from the
   [releases page](https://github.com/xiaodoudou/PlexIPTV/releases/latest).
2. `chmod +x PlexIPTV.macos.x64`, then run it once. It writes a
   `settings.json` beside itself.
3. Put your m3u URL in `m3u8.remote` and run it again.
4. In Plex: Settings, Live TV & DVR, Set up Plex DVR, and if it is not found
   automatically use "Don't see your device?" and enter `127.0.0.1:1234`.

Gatekeeper will object the first time, since the binary is not notarised.
Right click, Open, and allow it once.

Recording works the same as any Plex DVR channel once the tuner is added.

Closing, but do open a new issue if a specific step fails.
